test/core/end2end/fixtures/h2_tls_certwatch_async_tls1_3.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <string.h>
+
+#include <string>
+
+#include <grpc/grpc.h>
+
+#include "src/core/lib/gprpp/global_config_generic.h"
+#include "src/core/lib/gprpp/host_port.h"
+#include "src/core/lib/security/security_connector/ssl_utils_config.h"
+#include "test/core/end2end/end2end_tests.h"
+#include "test/core/end2end/fixtures/h2_tls_common.h"
+#include "test/core/util/port.h"
+#include "test/core/util/test_config.h"
+
+static grpc_end2end_test_fixture
+chttp2_create_fixture_async_verifier_cert_watcher(const grpc_channel_args*,
+                                                  const grpc_channel_args*) {
+  grpc_end2end_test_fixture f;
+  int port = grpc_pick_unused_port_or_die();
+  fullstack_secure_fixture_data* ffd = new fullstack_secure_fixture_data();
+  memset(&f, 0, sizeof(f));
+  ffd->localaddr = grpc_core::JoinHostPort("localhost", port);
+  SetTlsVersion(ffd, SecurityPrimitives::TlsVersion::V_12);
```

Comment:
Should this be `TlsVersion::V_13`?

Reply:
It probably should, but it'd be a behavior change:
https://github.com/grpc/grpc/pull/30569/files#diff-65bfaec96ba210dbb95add74a6d0541c1c827542c030e78c86806efbd88021c4L238

Reply:
Ok, probably best not to change it now. Please leave a strongly-worded TODO to fix that or explain that if it's intentional

Reply:
I'm gonna follow this up with a PR to fix it that we can sort out later.